ibridges/search.py

```diff
@@ -0,0 +1,54 @@
+""" Data query
+"""
+from typing import Optional
+
+from ibridges.irodsconnector import keywords as kw
+from ibridges import Session
+
+
+def search(session: Session, path: Optional[str] = None, checksum: Optional[str] = None,
```

Comment:
I think it would be easier for users if it gave back a list of dictionaries or named tuples, instead of plain tuples.